src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,73 @@
-export const Pagination = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage?: number;
+  onPageChange: (value: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}) => {
+  const perPageSelectors: number =
+    total % Number(perPage) ? Math.floor(total / perPage) + 1 : total / perPage;
```

Comment:
The name `perPageSelectors` is a bit misleading since it suggests UI elements rather than a numeric value. Consider renaming it to something more indicative of its purpose, like `totalPages`.